docs/static/monitoring/monitoring-ea.asciidoc

```diff
@@ -1,85 +1,78 @@
-configure all required settings:
-* Under **Collect Logstash application and slowlog logs**, modify the log paths
+. Configure the integration to collect logs.
+* Make sure that **Logs** is turned on if you want to collect logs from your {ls} instance, ensuring that the required settings are correctly configured:
+* Under **Logs**, modify the log paths
 to match your {ls} environment.
-* Under **Collect Logstash node metrics and stats**, make sure the hosts setting
+
+. Configure the integration to collect metrics
+* Make sure that **Metrics (Stack Monitoring)** is turned on, and **Metrics (Technical Preview)** is turned off, if you
+want to collect metrics from your Logstash instance
+* Under **Metrics (Stack Monitoring)**, make sure the hosts setting
 points to your {ls} host URLs. By default, the integration collects {ls}
 monitoring metrics from `localhost:9600`. If that host and port number are not
 correct, update the `hosts` setting. If you configured {ls} to use encrypted
 communications, you must access it via HTTPS. For example, use a `hosts` setting
 like
 `https://localhost:9600`.
+
+. Choose where to add the integration policy. +
+Click **New hosts** to add it to new agent policy or **Existing hosts** to add it to an existing agent policy.
+. In the popup, click **Add {agent} to your hosts** to open the **Add agent**
+flyout.
 +
-**Elastic security.** The Elastic {security-features} are enabled by default. 
-Expand **Advanced options** and enter the username and password of a user that has
-the `remote_monitoring_collector` role.
-. Choose where to add the integration policy. Click **New hosts** to add it to
-new agent policy or **Existing hosts** to add it to an existing agent policy.
-. Click **Save and continue**. This step takes a minute or two to complete. When
-it's done, you'll have an agent policy that contains an integration for
-collecting monitoring data from {kib}.
-. If an {agent} is already assigned to the policy and deployed to the host where
-{kib} is running, you're done. Otherwise, you need to deploy an {agent}. To
-deploy an {agent}:
-.. Go to **{fleet} -> Agents**, then click **Add agent**.
-.. Follow the steps in the **Add agent** flyout to download, install,
-and enroll the {agent}. Make sure you choose the agent policy you created
-earlier.
-. Wait a minute or two until incoming data is confirmed.
-. {kibana-ref}/monitoring-data.html[View the monitoring data in {kib}].
\ No newline at end of file
+TIP: If you accidentally close the popup, go to **{fleet} > Agents**, then click
+**Add agent** to access the flyout.
+
+[discrete]
+[[add-agent-to-fleet-ea]]
+==== Install and run an {agent} on your machine
+// Include section about installing agents
+include::monitoring-install.asciidoc[tag=install-agent-widget]
+
+[discrete]
+[[confirm-ea]]
+=== Confirm Enrollment
+// Include section about confirming enrollment
+include::monitoring-confirm.asciidoc[tag=confirm-enrollment-widget]
```

Comment:
The heading is "Confirm Enrollment," but the section leads off with "After enrollment is confirmed..."  This makes me think that "Confirm Enrollment" should be part of the previous section, and that we need a more appropriate name for what's going on in this section.